server/src-lib/Hasura/Server/Init.hs

```diff
@@ -943,6 +945,7 @@ serveOptsToLog so =
       , "log_level" J..= soLogLevel so
       , "plan_cache_options" J..= soPlanCacheOptions so
       , "websocket_compression_options" J..= show (WS.connectionCompressionOptions . soConnectionOptions $ so)
+      , "websockets_keep_alive" J..= show (soWebsocketKeepAlive so)
```

Comment:
this should also be `websocket_keep_alive` :)

Reply:
omg what else did I forgot :)